protocol/app/process/prices/expected_keepers.go

```diff
@@ -0,0 +1,15 @@
+package prices
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	pricestypes "github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
+)
+
+// PricesKeeper defines the expected Prices keeper used for `DefaultMarketPriceDecoder`.
+type PricesKeeper interface {
```

Comment:
This is basically already defined in process https://github.com/dydxprotocol/v4-chain/blob/83dd2c9df394351acd6340077cdb527661b5d1af/protocol/app/process/expected_keepers.go#L16

Adding more packages here seems to be creating a lot of churn/duplication.